axonserver/src/main/java/io/axoniq/axonserver/metric/BaseMetricName.java

```diff
@@ -57,24 +57,12 @@ public enum BaseMetricName implements MetricName {
      * Metric for the rate of snapshots stored (tags: context)
      */
     AXON_SNAPSHOTS("axon.snapshot", "Number of snapshots stored"),
-    AXON_GLOBAL_SUBSCRIPTION_TOTAL("axon.GlobalSubscriptionMetricRegistry.total",
-                                   "Total number of subscription queries subscribed"),
-    AXON_GLOBAL_SUBSCRIPTION_UPDATES("axon.GlobalSubscriptionMetricRegistry.updates",
-                                     "Total number of updates submitted on subscription queries"),
-    AXON_GLOBAL_SUBSCRIPTION_ACTIVE("axon.GlobalSubscriptionMetricRegistry.active",
-                                    "Active number of subscription queries subscribed"),
-    AXON_QUERY_SUBSCRIPTION_TOTAL("axon.QuerySubscriptionMetricRegistry.total",
-                                  "Total number of subscription queries subscribed on this node"),
-    AXON_QUERY_SUBSCRIPTION_UPDATES("axon.QuerySubscriptionMetricRegistry.updates",
-                                    "Total number of updates submitted on subscription queries on this node"),
-    AXON_QUERY_SUBSCRIPTION_ACTIVE("axon.QuerySubscriptionMetricRegistry.active",
-                                   "Active number of subscription queries on this node"),
-    AXON_APPLICATION_SUBSCRIPTION_TOTAL("axon.ApplicationSubscriptionMetricRegistry.total",
-                                        "Total number of subscription queries subscribed per application"),
-    AXON_APPLICATION_SUBSCRIPTION_UPDATES("axon.ApplicationSubscriptionMetricRegistry.updates",
-                                          "Total number of updates submitted on subscription queries per application"),
-    AXON_APPLICATION_SUBSCRIPTION_ACTIVE("axon.ApplicationSubscriptionMetricRegistry.active",
-                                         "Active number of subscription queries on this node per application"),
+    AXON_SUBSCRIPTION_TOTAL("axon.subscriptionquery.total",
```

Comment:
this is not backwards compatible change

Reply:
Added the old metrics again. To avoid duplicate metrics, you can set the property "axoniq.axonserver.legacy-metrics-enabled" to false. 
Updated the names for the new metrics:
axon.queries.subscriptionquery.throughput.total
axon.queries.subscriptionquery.throughput.updates
axon.queries.subscriptionquery.duration
axon.queries.subscriptionquery.saturation.active